dev-support/test-patch.sh

```diff
@@ -1039,6 +1058,8 @@ function parse_args
 
   GITDIFFLINES="${PATCH_DIR}/gitdifflines.txt"
   GITDIFFCONTENT="${PATCH_DIR}/gitdiffcontent.txt"
+  GITUNIDIFFLINES="${PATCH_DIR}/gitdiffunilines.txt"
```

Comment:
shellcheck: 15: note: Double quote to prevent globbing and word splitting. [SC2086]